src/Mod/CAM/libarea/clipper.cpp

```diff
@@ -4832,10 +4832,10 @@ inline double DistanceSqrd(const IntPoint& pt1, const IntPoint& pt2)
 double DistanceFromLineSqrd(const IntPoint& pt, const IntPoint& ln1, const IntPoint& ln2)
 {
     // The equation of a line in general form (Ax + By + C = 0)
-    // given 2 points (x�,y�) & (x�,y�) is ...
-    //(y� - y�)x + (x� - x�)y + (y� - y�)x� - (x� - x�)y� = 0
-    // A = (y� - y�); B = (x� - x�); C = (y� - y�)x� - (x� - x�)y�
-    // perpendicular distance of point (x�,y�) = (Ax� + By� + C)/Sqrt(A� + B�)
+    // given 2 points (x�,y�) & (x�,y�) is ...
+    //(y� - y�)x + (x� - x�)y + (y� - y�)x� - (x� - x�)y� = 0
+    // A = (y� - y�); B = (x� - x�); C = (y� - y�)x� - (x� - x�)y�
+    // perpendicular distance of point (x�,y�) = (Ax� + By� + C)/Sqrt(A� + B�)
```

Comment:
Very annoying that vscode does this. Need to figure out how to disable this auto-formatting.